src/net.cpp

```diff
@@ -1512,16 +1512,10 @@ void CConnman::SocketHandler()
             }
         }
 
-        //
-        // Send
-        //
-        if (sendSet)
-        {
-            LOCK(pnode->cs_vSend);
-            size_t nBytes = SocketSendData(pnode);
-            if (nBytes) {
-                RecordBytesSent(nBytes);
-            }
+        if (sendSet) {
+            // Send data
+            size_t bytes_sent = WITH_LOCK(pnode->cs_vSend, return SocketSendData(pnode));
```

Comment:
same commit: The lock annotation for this member function `SocketSendData` is in the wrong place. Should be in the h file not the cpp file because otherwise moving code can disable the annotation.

Would be nice to fix in this pull and maybe also update the `SocketSendData` pointer arg to a reference?

Reply:
Done.